external/fv3fit/fv3fit/pytorch/cyclegan/cyclegan_trainer.py

```diff
@@ -61,7 +64,7 @@ class CycleGANNetworkConfig:
     discriminator_weight: float = 1.0
 
     def build(
-        self, n_state: int, n_batch: int, state_variables, scalers
+        self, n_state: int, nx: int, ny: int, n_batch: int, state_variables, scalers
```

Comment:
Sorry, this change belongs to the "[add GeographicBias layer to CycleGAN](https://github.com/ai2cm/fv3net/pull/2051/commits/86d9c8db89a2cffe5c340f98914d870a86cf80d2)" commit